### DISCUSSION TITLE
Translation: Integration of Crowdin for Translation Management

### DISCUSSION THREAD

**Discussion by Tiberth** [zoonk/uneebee#178] [Ideas]
I would like to propose the integration of Crowdin to manage translations within the project. This integration could extend beyond the program itself to include any associated courses in the future. While I acknowledge that enabling it for all courses might pose challenges or may not align with your mission, I believe leveraging Crowdin for the program itself would offer a user-friendly platform for collaborative translation work, simplifying the management of multiple languages.

Thank you for taking the time to consider this proposal, and I want to convey my gratitude for your efforts in developing this program.

**Comment by ceolinwill** [comment]
Thank you for your proposal, @Tiberth. I love Crowdin but that's not a viable option at the moment for financial reasons:

1. I can't afford to pay a license at the moment. I don't have any [sponsors](https://github.com/sponsors/wceolin) and @zoonk is pre-revenue.
2. Crowdin has a [free plan](https://crowdin.com/pricing) but it has limitations. I don't want to reach a point where I'm above the free tier and I must pay for it.
3. Crowdin has an [open-source license](https://crowdin.com/page/open-source-project-setup-request) but it's not valid if we have commercial products related to the open-source project. I'm going to develop three commercial products on top of it: for [businesses](https://wikaro.com), [schools](https://educasso.com) and [creators](https://mywisek.com).

Once one of those products start having revenue, I may reconsider this and add a Crowdin integration.
